pangeo-notebook/binder/environment.yml

```diff
@@ -56,6 +58,3 @@ dependencies:
   # xarray-related
   - xgcm
   - xrft
-  - pip:
-    - regionmask
-    - rio-cogeo
```

Comment:
Thanks for doing this @ocefpaf. One quick request? This image inherits from the base-notebook image. Can you remove the pip installs there too?

Reply:
> One quick request? This image inherits from the base-notebook image. Can you remove the pip installs there too?

Sure. Can you point me where that is? I'm still familiarizing myself with the pangeo repos. (Also, I'm planning to add the `strict` channel option and add the defaults channel back but that must be done on the Dockerfile. See my explanation in https://github.com/pangeo-data/pangeo-cloud-federation/pull/305.)

Reply:
see https://github.com/pangeo-data/pangeo-stacks/blob/master/base-notebook/binder/environment.yml#L16

Reply:
Done.

PS: where is the canonical place for the Dockerfile? I would like to add the `strict` option and maybe `pip` in the conda root env so the `%pip` magic works in the notebooks.

Reply:
Thanks @ocefpaf - the docker file addition will need to some special consideration so let's deal with that separately.

Reply:
Yep. I'm planning 2-3 PRs to implement everything in a controlled way.